subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -17,6 +17,8 @@
 
 This chapter provides the information you need to migrate your older Gradle 4.x builds to Gradle 5.0. In most cases, you will need to apply the changes from all versions that come after the one you're upgrading from. For example, if you're upgrading from Gradle 4.3, you will also need to apply the changes since 4.4, 4.5, etc.
 
+TIP: If you are using Android Studio, you need to move to version 3.3 or higher of both the Android Gradle Plugin and Android Studio.
```

Comment:
"If you are using Android Studio" -> "If you are using Gradle for Android"
Because it will not only affect Studio users, but also command line ones that may not use the IDE.

Reply:
And the version should even be `3.4` given the latest reports we had. If Google manages to fix the problem for `3.3`, we can always advertise that later.

Reply:
updated.